<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 89, 150)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="argument1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(253, 253, 253)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="argument2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(253, 253, 253)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font: 14pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>Введіть аргументи:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font: 14pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font: 14pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font: 14pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>                                         Тут буде результат</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font:  14 pt, &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>Знайти</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
